docker-awscli.tf

```diff
@@ -0,0 +1,65 @@
+resource "github_repository" "docker_awscli" {
+  name             = "docker-awscli"
+  description      = "Docker container for awscli. Includes a file to source at `/assumerole` which expects to be provided `AWS_ROLE_ARN` env var and will export `AWS_SECRET_ACCESS_KEY`, `AWS_ACCESS_KEY_ID` and `AWS_SESSION_TOKEN` env vars."
+  auto_init        = true
+
```

Comment:
nitpick: if you remove this blank line then the alignment of `=` improves for the following 2 lines :)